### PR TITLE
Clarify the comment in agent Dockerfile.

### PR DESCRIPTION
**Pull request by darobs**
Change the comment in Dockerfile to give the technical reason for the addition.

### CODE REVIEW

File: edge-agent/docker/linux/amd64/Dockerfile [hunk 1]
```diff
@@ -9,7 +9,7 @@ ARG EXE_DIR=.
 RUN apk update && \
     apk add --no-cache snappy
 
-# create a user to allow agent to optionally run as non-root
+# Create a user to allow installing the trustbundle under a user context
```
Comment (varunpuranik) [inline edge-agent/docker/linux/amd64/Dockerfile:12]: What do you mean by installing trust bundle under a user context? Where does the trustbundle get installed?

Comment (darobs) [reply edge-agent/docker/linux/amd64/Dockerfile:12]: We set the X509store location to `CurrentUser` - which creates the store in the current user's directory. If agent is running as root, it uses `/root` as the directory, this creates a user for installing. For Edge on K8s, we can run Agent as non-root, so we create this user for a place to put the X509 store.

File: edge-agent/docker/linux/arm64v8/base/Dockerfile [hunk 2]
```diff
@@ -13,7 +13,7 @@ RUN apt-get update && \
     ln -s /lib/aarch64-linux-gnu/libdl.so.2 /usr/lib/aarch64-linux-gnu/libdl.so && \
     ln -s /lib/aarch64-linux-gnu/libc.so.6 /usr/lib/aarch64-linux-gnu/libc.so.6
 
-# create a user to allow agent to optionally run as non-root
+# Create a user to allow installing the trustbundle under a user context
```
Comment (CindyXing) [inline edge-agent/docker/linux/arm64v8/base/Dockerfile:16]: This was needed for K8s. I thought we planned to clarify it.

Comment (CindyXing) [reply edge-agent/docker/linux/arm64v8/base/Dockerfile:16]: Given we'll be using the non-root user for single device, this comment without k8s specified is ok now.

**Comment by darobs** [comment]
Since we've committed to allow agent to run as non-root in 1.0.10, this comment update is not necessary,

**Pull request closed without merging**